metadata-io/src/main/java/com/linkedin/metadata/graph/elastic/ESGraphWriteDAO.java

```diff
@@ -50,8 +51,9 @@ public BulkByScrollResponse deleteByQuery(@Nullable final String sourceType, @No
       @Nullable final String destinationType, @Nonnull final Filter destinationEntityFilter,
       @Nonnull final List<String> relationshipTypes, @Nonnull final RelationshipFilter relationshipFilter) {
     BoolQueryBuilder finalQuery =
-        buildQuery(sourceType, sourceEntityFilter, destinationType, destinationEntityFilter, relationshipTypes,
-            relationshipFilter);
+        buildQuery(sourceType == null ? ImmutableList.of() : ImmutableList.of(sourceType), sourceEntityFilter,
```

Comment:
you can also use Collections.emptyList for an empty list : ) 

also, this method can take null right? 

if it shouldn't take null values  you can mark the param as @Nonnull